.travis.yml

```diff
@@ -3,8 +3,19 @@ node_js: "8"
 dist: trusty
 cache:
   yarn: true
+  directories:
+  - "$HOME/gcloud"
 install:
   - yarn
+  - if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then mkdir -p $HOME/gcloud && wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud && cd $HOME/gcloud && tar xzf google-cloud-sdk.tar.gz && printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh && cd $TRAVIS_BUILD_DIR;
```

Comment:
Looks like you can set `CLOUDSDK_CORE_DISABLE_PROMPTS=1` so you don't need the `y` trick.

https://stackoverflow.com/a/38776394/1373746